packages/fast-components-react-base/src/listbox-item/listbox-item.schema.json

```diff
@@ -0,0 +1,35 @@
+{
+    "$schema": "http://json-schema.org/schema#",
+    "title": "Listbox item",
+    "description": "A listbox item component's schema definition.",
+    "type": "object",
+    "id": "listbox-item",
+    "properties": {
+        "disabled": {
+            "title": "Disabled",
+            "type": "boolean"
+        },
+        "displayString": {
+            "title": "Display string",
+            "type": "string"
+        },
+        "id": {
+            "title": "ID",
```

Comment:
We have a few different ways we enumerate "ID" as a title - @nalogiudice do you recall what our decided format was here?

Reply:
HTML id attribute

Reply:
"Unique ID" seems most common, although there are other variants.

Reply:
(Gonna go with that, if we want to standardize we should open a bug for someone to do a consistency pass)

Reply:
I'm fine with this ^^^. I checked three different components and somehow the schemas were all different (although I think we did align these at some point?)